opencti-platform/opencti-graphql/src/schema/stixDomainObject.ts

```diff
@@ -97,7 +97,8 @@ const STIX_DOMAIN_OBJECT_LOCATIONS: Array<string> = [
 ];
 schemaTypes.register(ENTITY_TYPE_LOCATION, STIX_DOMAIN_OBJECT_LOCATIONS);
 export const isStixDomainObjectLocation = (type: string): boolean => {
-  return R.includes(type, STIX_DOMAIN_OBJECT_LOCATIONS) || type === ENTITY_TYPE_LOCATION;
+  const locationEntities = schemaTypes.get(ENTITY_TYPE_LOCATION);
+  return R.includes(type, locationEntities) || type === ENTITY_TYPE_LOCATION;
```

Comment:
You can replace Ramda here by https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/includes